doc/audit/builtin_maps.rst

```diff
@@ -0,0 +1,29 @@
+Built-in Maps
+=============
+
+`public:ccf.gov.nodes.code_ids`
+-------------------------------
+
+This table contains all the versions of the code allowed to join the current network.
+
+Key
+~~~
+
+base64 string representation of MRENCLAVE
+
+Value
+~~~~~
+
+.. doxygenenum:: ccf::CodeStatus
+   :project: CCF
+
+Example
+~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+
+   * - Code ID
+     - Status
+   * - `cae46d1...bb908b64e`
+     - `ALLOWED_TO_JOIN`
```

Comment:
What does this look like?

Reply:
![image](https://user-images.githubusercontent.com/4016369/101811884-8f074300-3b12-11eb-87ab-e42d2f36c9aa.png)

Reply:
It's not necessarily the final word on how we want to document the KV, but it's a start.